app/src/main/java/com/himanshoe/sample/MainActivity.kt

```diff
@@ -13,9 +13,12 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         setContent {
             Kalendar(
-                kalendarEvents = listOf(KalendarEvent(LocalDate.now().plusDays(3), "", "")),
+                kalendarEvents = listOf(
+                    KalendarEvent(LocalDate.now().plusDays(3), "Test event one", "Test one description"),
+                    KalendarEvent(LocalDate.now().plusDays(3), "Test event two", "Test two description")
+                ),
                 kalendarKonfig = KalendarKonfig(weekCharacters = 2),
-                onCurrentDayClick = { date, event ->
+                onCurrentDayClick = { date, eventsList ->
```

Comment:
can we make it events in place of eventsList

Reply:
Done